src/app/clusters/occupancy-sensor-server/occupancy-sensor-server.h

```diff
@@ -17,8 +17,76 @@
 
 #pragma once
 
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
 #include <app/util/af-types.h>
 #include <app/util/basic-types.h>
+#include <app/util/config.h>
+#include <lib/core/DataModelTypes.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace OccupancySensing {
+
+/**
+ * Interface to help manage the Hold Time Limits of the Occupancy Sensing Cluster.
+ */
+class HoldTimeLimitsManager : public AttributeAccessInterface
```

Comment:
The HoldTimeLimitsManager should not have `AttributeAccessInterface`, but should be business-logic only.

There needs to be a separate class for the OccupancySensing AttributeAccessInterface.

Reply:
Changed in new code, please check